addon/components/nrg-application.js

```diff
@@ -1,59 +1,70 @@
-import Component from '@ember/component';
-import { computed } from '@ember/object';
-import { alias, and, reads } from '@ember/object/computed';
+import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { htmlSafe } from '@ember/string';
-import ResizeMixin from 'ember-nrg-ui/mixins/resize';
-import layout from '../templates/components/nrg-application';
-
-export default Component.extend(ResizeMixin, {
-  layout,
-
-  application: service(),
-
-  lightbox: service(),
-
-  modal: service(),
-
-  responsive: service(),
-
-  flashMessages: service(),
-
-  sidebarIsOpen: false,
-
-  sidebarMenuIsOpen: alias('application.sidebarMenuIsOpen'),
-
-  classNames: ['nrg-application'],
-
-  classNameBindings: ['fullscreenMap:fullscreen-map', 'computerScreenSidebarActive:large-screen-sidebar-active'],
-
-  isComputerScreen: alias('responsive.isComputerScreenGroup'),
-
-  title: reads('application.pageTitle'),
-
-  init() {
-    this._super(...arguments);
-    this.set('sidebarMenuIsOpen', true);
-  },
-
-  computerScreenSidebarActive: and('isComputerScreen', 'sidebarMenuIsOpen'),
-
-  mainContentStyle: computed('screenHeight', function() {
-    return htmlSafe(`height:${this.screenHeight}px`);
-  }),
-
-  actions: {
-    toggleSidebar() {
-      if (this.isComputerScreen) {
-        this.toggleProperty('sidebarMenuIsOpen');
-      } else {
-        this.toggleProperty('sidebarIsOpen');
-      }
-    },
-    clickedLink(item) {
-      if (this.clickedSidebarItem) {
-        this.clickedSidebarItem(item);
-      }
-    },
-  },
-});
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+export default class NrgFormComponent extends Component {
+  @service
+  application;
+
+  @service
+  lightbox;
+
+  @service
+  modal;
+
+  @service
+  responsive;
+
+  @service
+  resize;
+
+  @service
+  flashMessages;
+
+  @tracked
+  sidebarIsOpen = false;
+
+  @tracked
+  mainContentStyle;
+
+  get computerScreenSidebarActive() {
+    return (
+      this.responsive.isComputerScreenGroup &&
+      this.application.sidebarMenuIsOpen
+    );
+  }
+
+  constructor() {
+    super(...arguments);
+    this.setMainContentStyle();
+  }
+
+  setMainContentStyle() {
+    return htmlSafe(`height:${window.innerHeight}px`);
+  }
+
+  @action
+  onInsert() {
+    this.resize.on('didResize', this, this.setMainContentStyle);
+  }
+
+  @action
+  onDestroy() {
+    this.resize.off('didResize', this, this.setMainContentStyle);
+  }
+
+  @action
+  toggleSidebar() {
+    if (this.responsive.isComputerScreenGroup) {
+      this.application.sidebarMenuIsOpen = !this.application.sidebarMenuIsOpen;
+    } else {
+      this.sidebarIsOpen = !this.sidebarIsOpen;
+    }
```

Comment:
```suggestion
    let property = 'sidebarIsOpen';
    if (this.responsive.isComputerScreenGroup) {
      property = 'application.sidebarMenuIsOpen';
    }
    this.toggleProperty(property);
```

Reply:
`toggleProperty` does not exist in octane components:
![image](https://user-images.githubusercontent.com/2756533/142034148-97c12c40-9104-48f9-8a33-f7be9e1bda16.png)

Reply:
Ah my bad, I read the docs for the Ember component that still has them, not the Glimmer component.